src/fastapi_poe/types.py

```diff
@@ -24,6 +24,20 @@ class MessageFeedback(BaseModel):
     reason: Optional[str]
 
 
+class CostItem(BaseModel):
+    """
+
+    An object representing a cost item used for authorization and charge request.
+    #### Fields:
+    - `amount_usd_milli_cents` (`int`)
+    - `description` (`str`)
+
+    """
+
+    amount_usd_milli_cents: int
```

Comment:
do we want to expose this term of milli cents to users? Or should we stick to using message points?

Reply:
Any user facing amounts should still be in compute points but the creator facing earnings should use USD as this is what the creators will be paid out (milli cents to avoid using float for monetary amounts).